src/mono/browser/runtime/pinvoke.c

```diff
@@ -48,9 +48,9 @@ void*
 wasm_dl_get_native_to_interp (const char *key, void *extra_arg)
 {
 #ifdef GEN_PINVOKE
-	for (int i = 0; i < sizeof (wasm_native_to_interp_map) / sizeof (void*); ++i) {
-		if (!strcmp (wasm_native_to_interp_map [i], key)) {
-			void *addr = wasm_native_to_interp_funcs [i];
+	for (int i = 0; wasm_native_to_interp_table [i].name != NULL; ++i) {
```

Comment:
are we sure that the last slot is always NULL ?

Reply:
yes, the corresponding change to the generator explicitly adds a trailing `{NULL, NULL}`

Reply:
👍 I'll describe the reasoning once I've finished tweaking it